scripts/setup/autoenv.sh

```diff
@@ -29,12 +29,12 @@ fi
 
 
 if [ -z "${HOLBA_DIR}" ]; then
-  HOLBA_DIR=${SETUP_DIR}/../..
+  HOLBA_DIR="${SETUP_DIR}/../.."
   HOLBA_DIR=$(readlink -f "${HOLBA_DIR}")
 fi
 
 if [[ ! -d "${HOLBA_DIR}/scripts/setup" ]]; then
```

Comment:
What about adding your README.md head, grep, "HolBA" check also?

Reply:
Which one do you prefer?
```bash
head -n1 README.md | grep -c '# HolBA'
```

or

```bash
git remote -v | grep -c 'github.com:kth-step/HolBA'
```

Reply:
Hmm, both have advantages and disadvantages. We could add both for now so that we have them and make it robust now. And then we would remove one or adjust it as soon as it is required. What do you think about this cannonball solution?

Reply:
Did both + the current one, so 3 tests in total :)